src/xrCore/xrCore.h

```diff
@@ -102,7 +102,7 @@
 #endif
 
 #include <time.h>
-#define MODULE_NAME "xrCore.dll"
+#define MODULE_NAME "xrCore"
```

Comment:
@Kaffeine It is. You could just read the [specification on MSDN](https://msdn.microsoft.com/en-us/library/windows/desktop/ms684175%28v=vs.85%29.aspx).